### PR TITLE
fix the event length issue

### PR DESCRIPTION
**Pull request by gram-incolo**
fixes the constraint of having event length > 10 and adds event creator's name to the recipe name

### CODE REVIEW

**Comment by kjawadDeveloper2** [comment]
Kindly make your dart code length 120 instead of 80.

File: evently/lib/models/events.dart [hunk 1]
```diff
@@ -216,17 +234,22 @@ class PerksModel {
...
+  factory PerksModel.updateName(
+          {required String name, required PerksModel perksModel}) =>
+      PerksModel(
         name: name,
         description: perksModel.description,
       );
 
-  factory PerksModel.updateDescription({required String description, required PerksModel perksModel}) => PerksModel(
+  factory PerksModel.updateDescription(
+          {required String description, required PerksModel perksModel}) =>
```
Comment (kjawadDeveloper2) [inline evently/lib/models/events.dart:245]:
```suggestion
          {required String description, required PerksModel perksModel,}) =>
```

File: evently/lib/models/events.dart [hunk 2]
```diff
@@ -216,17 +234,22 @@ class PerksModel {
...
+      PerksModel(
         name: perksModel.name,
         description: description,
       );
 
-  factory PerksModel.fromJson(Map<String, dynamic> map) => PerksModel(name: map['name'] ?? '', description: map['description'] ?? '');
+  factory PerksModel.fromJson(Map<String, dynamic> map) => PerksModel(
+      name: map['name'] ?? '', description: map['description'] ?? '');
```
Comment (kjawadDeveloper2) [inline evently/lib/models/events.dart:252]:
```suggestion
      name: map['name'] ?? '', description: map['description'] ?? '',);
```

**Comment by gram-incolo** [comment]
@kjawadDeveloper2 I believe 80 is a good standard which keeps things easy to read.

**Review by MikeSofaer — APPROVED** [review]

**Pull request merged**